### PR TITLE
RFR: 8265667: Add README.md and CONTRIBUTING.md to FX 11-dev/rt repo

### PR DESCRIPTION
**Pull request by kevinrushforth**
This Draft PR is to allow review of the proposed README.md and CONTRIBUTING.md files for jfx 11 updates. See the following JBS bug:

[JDK-8265667](https://bugs.openjdk.java.net/browse/JDK-8265667): Add README.md and CONTRIBUTING.md to FX 11-dev/rt repo

NOTE: this PR will not be integrated. Instead, once approved, it will be pushed to the [openjfx/11-dev/rt](https://hg.openjdk.java.net/openjfx/11-dev/rt) HG repo.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+Contributing to OpenJFX 11 Updates
+==================================
+
+The OpenJFX 11 Updates repository is _not_ accepting new contributions. If you are looking to contribute to OpenJFX, please visit [openjdk/jfx](https://github.com/openjdk/jfx) instead.
+
+Important bug fixes that have already been fixed in the mainline can be backported with approval. New fixes should first go into the mainline, and then can be considered for backporting. Approval from one of the project leads is required prior to integrating the fix.
+
+To backport a fix, you need to import the patch from mainline into a branch of your personal fork of this repo. Then, you can create the pull request. We strongly recommend creating a backport-style pull request, by creating the PR with the title:
+
```
Comment (johanvos) [inline CONTRIBUTING.md:9]: shouldn't this be a requirement instead of a recommendation? I don't think there is a reason why using that style would not be possible?

Comment (kevinrushforth) [reply CONTRIBUTING.md:9]: For fixes that are backports it should always be possible, so I'll reword this to make that clear.

Comment (kevinrushforth) [reply CONTRIBUTING.md:9]: Updated to make it clear that backport PRs are needed.

**Review by johanvos — APPROVED** [review]
